index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta property="og:title" content="Linkbrary" />
+    <meta property="og:url" content="https://weekly-mission-hsw.netlify.app/" />
+    <meta property="og:image" content="./images/logo.png" />
+    <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해보세요" />
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="./src/style.css" />
+</head>
+<body>
+    <nav>
+        <div class="gnb">
+            <a href="/">
+                <img calss="button-logo" src="./images/logo.png" alt="홈으로 연결된 Linkbrary 로고" />
+            </a>
+            <a class="button" href="signin.html">
+                <span>로그인</span>
```

Comment:
span 태그로 불필요하게 감싼 부분들이 보이네요.

```suggestion
                로그인
```